src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/AbstractEntityAIGuard.java

```diff
@@ -213,6 +213,7 @@ private void onCombatEnter()
     private void onCombatLeave()
     {
         worker.getNavigation().getPathingOptions().setCanUseRails(((EntityCitizen) worker).canPathOnRails());
+        worker.getNavigation().getPathingOptions().setCanClimbVines(((EntityCitizen) worker).canClimbVines());
```

Comment:
you want to disable this above in onCombatEnter() else the reenabling here has no purpose

Reply:
Ohh, I had just copied it for everywhere we do the rails stuff. I can remove this entirely.

Reply:
either way works xD